test-app/src/main/assets/readium/404.xhtml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
+    "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
+<head>
+    <title>Resource not found</title>
```

Comment:
It's exactly the same as the `404.xhtml` in the navigator module. I copied it as an example on how to override it in the test app.